cli/tests/test_cli.py

```diff
@@ -49,10 +48,10 @@ def test_raw_to_bids():
             mne_bids_cp.run()
 
 
-def test_cp():
+def test_cp(tmpdir):
     """Test mne_bids cp."""
-    output_path = _TempDir()
-    data_path = fetch_brainvision_testing_data()
```

Comment:
here, a `datasets.py` module dependency was unnecessary to begin with ... easy to remove